### ISSUE TITLE
Is it possible to show the suggestions when at least 1 character has been entered?

### ISSUE THREAD

**Issue by URSFR** [rohanjariwala03/drop_down_search_field#22]
I have thought about using the TextFieldConfiguration onChange but I don't know where to incorporate it to limit the number of results

**Comment by rohanjariwala03** [comment]
Can you explain bit more regarding what you're expecting.

**Comment by URSFR** [comment]
> Can you explain bit more regarding what you're expecting.

Sure. The function I am asking about is that the list of suggestions can appear only when the user inserts at least 1 character in the TextField and if it does not detect any character it closes automatically, like the way Google Maps Autocompleter works, which from from what is written it only shows suggestions but if you do not place anything, it does not show any suggestions. Is there currently a way to do it based on what is done in the package or would it have to be modified?

**Comment by rohanjariwala03** [comment]
I don't think so there is anything like this but if you want to add this feature then you can go ahead and make the PR else I can do it but there is no estimate time for this.
